### ISSUE TITLE
Copy all dependency version from gradle.properties to the extra properties of the target project

### ISSUE THREAD

**Issue by tlinkowski** [tlinkowski/tlinkowski-superpom#19]
Currently, `gradle.properties` looks like this:
https://github.com/tlinkowski/tlinkowski-superpom/blob/63cadd501e1211172b60a79a94fc34caf8878e75/gradle.properties#L18-L23

We could copy all the values below `#DEPENDENCIES` (or simply skip `version` and `group`) to the target project.

This would be useful for:
- storing once the version of Lombok
- storing once the versions of Spock+Groovy - they could be used e.g. here: https://github.com/tlinkowski/UniJ/blob/a48421d83e24b20b5755148c9acaa026511799cc/subprojects/unij-test/unij-test.gradle#L24-L25

In other words, versions of all the dependencies that could potentially be shared across many project would be defined in superPOM's `gradle.properties` and copied to all the other projects.

https://docs.gradle.org/current/userguide/kotlin_dsl.html#extra_properties

**Comment by tlinkowski** [comment]
More resources:
- https://github.com/gradle/gradle/blob/e9461fe3e2641398c6bdb82461ffbdadb21d0b1f/subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/ProjectExtensions.kt#L181
- https://github.com/gradle/gradle/blob/e9461fe3e2641398c6bdb82461ffbdadb21d0b1f/subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/PropertyDelegate.kt#L46
- https://github.com/gradle/gradle/blob/e9461fe3e2641398c6bdb82461ffbdadb21d0b1f/subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/ExtraPropertiesExtensions.kt#L41
